docs/integrations/ai-engines/monkeylearn.mdx

```diff
@@ -14,7 +14,10 @@ Create ML_ENGINE monkeylearn
 FROM monkeylearn
 ```
 
-![ML_engine](https://raw.githubusercontent.com/mindsdb/mindsdb/staging/docs/assets/tutorials/1.create_ml.png)
+<p align="center">
+  <img src="/assets/tutorials/1.create_ml.png" />
```

Comment:
Let's create a folder to store all images like `/assets/tutorials/monkeylearn/1.create_ml.png`

Reply:
i tried creating a folder but it got denied last night,i dont think i have authorization